doc/lua_api.md

```diff
@@ -5671,6 +5675,9 @@ Utilities
 * `minetest.urlencode(str)`: Encodes reserved URI characters by a
   percent sign followed by two hex digits. See
   [RFC 3986, section 2.3](https://datatracker.ietf.org/doc/html/rfc3986#section-2.3).
+* `minetest.time_to_day_night_ratio(time_of_day)`: Returns a "day-night ratio" value
```

Comment:
i think this should go somewhere in the env section. it feels wrong next to urlencode.

Reply:
Putting it into the "Environment access" section would be wrong since it doesn't use the environment. It's now directly below the color functions, perhaps that's better?